ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -156,6 +156,22 @@ MessageFormatFlags getOperationFlag() {
   }
 
   /**
+   * Get the total number of chunks for this operation.
+   * @return The total number of chunks.
+   */
+  int getNumChunksTotal() {
```

Comment:
Yes, I just added these for a test case.  I need a way of ensuring that chunks are read before `readInto()` is called.  Do you see any workarounds that don't require modifying `GetBlobOperation`?